### PR TITLE
Added link for contribution doc

### PR DESCRIPTION
**Pull request by hpandeycodeit**
Issue #204

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,10 @@ new features and changes.
 * [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-cli)
 * [Kubernetes Community site](http://kubernetes.io/community/)
 
+### Contribution
+
+Interested in contributing to Krew? Please refer to our [Contributing Guidelines](https://github.com/kubernetes-sigs/krew/blob/master/CONTRIBUTING.md) for more details
```
Comment (ahmetb) [inline README.md]: can you please follow the link convention at `## Documentation` section above,
also can you please do 80-character line wraps?

File: README.md [hunk 2]
```diff
@@ -155,6 +155,10 @@ new features and changes.
 * [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-cli)
 * [Kubernetes Community site](http://kubernetes.io/community/)
 
+### Contribution
```
Comment (ahmetb) [inline README.md]: `### Contributing` is probably a better title.

**Comment by ahmetb** [comment]
thanks for contributing!
left a few comments.
/approve

File: README.md (hunk 1 above)
Comment (hpandeycodeit) [reply README.md]: I have changed it as per the link convention

File: README.md (hunk 2 above)
Comment (hpandeycodeit) [reply README.md]: changed this as well

File: README.md (hunk 1 above)
Comment (ahmetb) [reply README.md]: looks like you missed the 80-character line wrap part :)

Comment (hpandeycodeit) [reply README.md]: somehow it didn't reflect in the changes. I have wrapped it now. :)

**Comment by corneliusweig** [comment]
@hpandeycodeit Thanks for spotting this!

Rather a question than a comment: should repo links in general refer to `sigs.k8s.io/krew` or is `github.com/kubernetes-sigs/krew` fine? (I'm aware that `krew-index` needs to point to github for now).

**Comment by ahmetb** [comment]
/lgtm
/approve

one nit: missing trailing period. :)

> should repo links in general refer to `sigs.k8s.io/krew` or is `github.com/kubernetes-sigs/krew` fine?

I'm indifferent about this. Both seem to work fine although GitHub makes it clear to anyone who's about to click a link that they're going to be going to GitHub.

I also own `krew.dev` domain which works the same way. :)

**Pull request merged**